containers/unit-tests/Dockerfile

```diff
@@ -1,38 +1,22 @@
-FROM ubuntu:16.04
+ARG debian_arch=amd64
 
-ARG arch=amd64
+FROM ${debian_arch}/ubuntu:19.04
 
-# dependencies which must be installed for the target architecture
-# we must do cross-builds on i386 as phantomjs is not available for i386
-ARG _crossdeps="libglib2.0-dev libgudev-1.0-dev libjson-glib-dev libkeyutils-dev liblvm2-dev libnm-glib-dev \
-    libpam0g-dev libpcp3-dev libpcp-import1-dev libpcp-pmda3-dev libpolkit-agent-1-dev libpolkit-gobject-1-dev \
-    libssh-dev libsystemd-dev libkrb5-dev libgnutls28-dev \
-    glib-networking glib-networking-dbg libc6-dbg libglib2.0-0-dbg pkg-config"
+ARG personality=linux64
+ADD setup.sh /
+RUN ${personality} /setup.sh ${personality}
```

Comment:
The script itself wants to run under the personality wrapper, plus it needs to know which personality it is so that it can write the /entrypoint script to ensure that everything else also runs under the same personality wrapper.